codegen/src/test/resources/software/amazon/awssdk/codegen/poet/builder/test-client-builder-class.java

```diff
@@ -81,6 +88,10 @@ protected final String signingName() {
         return "json-service";
     }
 
+    private JsonEndpointProvider defaultEndpointProvider() {
+        return JsonEndpointProvider.defaultProvider();
+    }
```

Comment:
nit: do we need a separate method for this?

Reply:
It follows the current patterns :). Don't think it's needed, but since it's here already I'll just keep it